analysis_scripts/analyze_quote_data/gender_analysis_all_years.md

```diff
@@ -0,0 +1,130 @@
+    quote_file = paste(proj_dir, 
+                    "/data/scraped_data/quote_table_raw_", curr_year, ".tsv", 
+                    sep="")
+    quote_df = read_corenlp_quote_files(quote_file)
+    quote_df$year = curr_year
+
+    full_quote_df = rbind(full_quote_df, quote_df)
+}
+full_quote_df = full_quote_df[-1,]
+
+head(full_quote_df)
+```
+
+    ##              file_id       est_speaker est_gender canonical_speaker
+    ## 2 news.2010.122.html    Aaron Krochmal       MALE    Aaron Krochmal
+    ## 3       463722a.html Aaron Ciechanover       MALE        biochemist
+    ## 4       463722a.html Aaron Ciechanover       MALE        biochemist
+    ## 5 news.2010.122.html    Aaron Krochmal       MALE    Aaron Krochmal
+    ## 6 news.2010.370.html       Abass Alavi       MALE       Abass Alavi
+    ## 7 news.2010.383.html            Abbott       MALE          Zambetti
+    ##        partial_name
+    ## 2    Aaron Krochmal
+    ## 3 Aaron Ciechanover
+    ## 4 Aaron Ciechanover
+    ## 5    Aaron Krochmal
+    ## 6       Abass Alavi
+    ## 7          Zambetti
+    ##                                                                                                                                                                                      quote
+    ## 2 "Although aspects of the findings contradict known behavioural and physiological work, the use of molecular genetic techniques is a new step in understanding how the facial pits work."
+    ## 3                                                           "It's the only programme that brings together people speaking so many different languages and gets them to work productively."
+    ## 4                                                                                                                                "It would be a terrible mistake to go in this direction,"
+    ## 5                                                             "The authors used elegant molecular techniques to confirm an idea long substantiated by physiological and behavioural data,"
+    ## 6                                                                     "This is a big lesson for all the major institutions that they are going to have to tighten their internal reviews."
+    ## 7                                                                                                                                                           "But it's a keystone finding,"
+    ##   year
+    ## 2 2010
+    ## 3 2010
+    ## 4 2010
+    ## 5 2010
+    ## 6 2010
+    ## 7 2010
+
+Now we join the tables together for comparison.
+
+``` r
+full_quote_df$is_benchmark = "full_data"
+bm_quote_df$is_benchmark = "benchmark"
+
+bm_full_quote_df = rbind(full_quote_df, bm_quote_df)
+```
+
+### compare benchmark and non-benchmark data
+
+<img src="gender_analysis_all_years_files/figure-markdown_github/unnamed-chunk-4-1.png" style="display: block; margin: auto;" />
+
+### compare proportions over all years
```

Comment:
Does the number of female quotes the same for 2019 and 2020?

Reply:
Also another question what is NEUTRAL gender?

Reply:
The number of female quotes are almost the same, but not quite. 2020: 691 Females, 2245 Male (23.5%) 2019: 706 Female, 2317 Male (23.27%). Good thing to check though!

Also, NEUTRAL is an output from coreNLP, when they assess a pronoun as neither male nor female. For example: their, they, we, it, etc. It's one of the four possible gender tags: HE, SHE, NEUTRAL, UNKNOWN. (https://nlp.stanford.edu/nlp/javadoc/javanlp-3.5.0/edu/stanford/nlp/dcoref/Dictionaries.Gender.html)